tests/D2L.CodeStyle.Analyzers.Test/UnsafeStatics/UnsafeStaticsAnalyzerTests.cs

```diff
@@ -672,6 +674,61 @@ internal class Bar {
 			AssertSingleDiagnostic( test, 7, 41, "foo", "test.Tests.Foo" );
 		}
 
+		[Test]
+		public void DocumentWithStaticField_ReadonlyUnsafeBaseClassWithUnsafeInitializer_Diag() {
+			const string test = @"
+	using System;
+	namespace test {
+		class Tests {
+			interface IUnsafe { void Magic(); } // could be anythinggggggg
+
+			class Safe : IUnsafe {
+				void IUnsafe.Magic() {} // looks safe to me
+				public static readonly Safe Instance { get; } = new Safe();
+			}
+
+			private readonly static IUnsafe foo = Safe.Instance;
+		}
+	}";
+
+			// Safe isn't sealed
+			AssertSingleDiagnostic( test, 12, 36, "foo", "test.Tests.Safe" );
+		}
+
+		[Test]
+		public void DocumentWithStaticField_ReadonlyUnsafeBaseClassWithSafeInitializer_NoDiag() {

```

Comment:
I think this test and the one before it should be named something like `WithSealedSafe` and `WithUnsealedSafe`; they both actually call the same initializer.